frontend/src/lib/ui/Button/ButtonPrimitives.scss

```diff
@@ -1,10 +1,16 @@
 :root {
+    --button-padding-x-xxs: 4px;
+    --button-padding-x-xs: 4px;
     --button-padding-x-sm: 4px;
     --button-padding-x-base: 6px;
     --button-padding-x-lg: 8px;
+    --button-padding-y-xxs: 3px;
+    --button-padding-y-xs: 3px;
     --button-padding-y-sm: 3px;
     --button-padding-y-base: 5px;
     --button-padding-y-lg: 7px;
+    --button-height-xxs: 20px;
+    --button-height-xs: 24px;
     --button-height-sm: 28px;
     --button-height-base: 30px;
     --button-height-base-tall: 45px;
```

Comment:
These new styles should be covered by UI snapshots i.e. stories, like LemonButton and kin